syntax = "proto3";
package even_connect.g2.efs_transmit;


enum eEvenFileServiceType {
    NOTIFICATION_JSON_WHITELIST = 0; // 1d54ef1
    ANDROID_MSG_JSON_NOTIFICATION = 1; // 1d54f11
    OTHER_FILE = 170; // 1d54ed1
}
enum eEvenFileServiceRsp {
    EVEN_FILE_SERVICE_RSP_SUCCESS = 0; // 1d54f71
    EVEN_FILE_SERVICE_RSP_START_ERR = 1; // 1d55011
    EVEN_FILE_SERVICE_RSP_DATA_CRC_ERR = 2; // 1d54fb1
    EVEN_FILE_SERVICE_RSP_FLASH_WRITE_ERR = 3; // 1d54ff1
    EVEN_FILE_SERVICE_RSP_TIMEOUT = 4; // 1d54f31
    EVEN_FILE_SERVICE_RSP_NO_RESOURCES = 5; // 1d54f91
    EVEN_FILE_SERVICE_RSP_RESULT_CHECK_FAIL = 6; // 1d54fd1
    EVEN_FILE_SERVICE_RSP_FAIL = 7; // 1d54f51
}
enum eEvenFileExportServiceCID {
    EVEN_FILE_SERVICE_CMD_EXPORT_START = 0; // 1d55051
    EVEN_FILE_SERVICE_CMD_EXPORT_DATA = 1; // 1d55071
    EVEN_FILE_SERVICE_CMD_EXPORT_RESULT_CHECK = 2; // 1d55031
}
enum eEvenFileSendServiceCID {
    EVEN_FILE_SERVICE_CMD_SEND_START = 0; // 1d550d1
    EVEN_FILE_SERVICE_CMD_SEND_DATA = 1; // 1d55091
    EVEN_FILE_SERVICE_CMD_SEND_RESULT_CHECK = 2; // 1d550b1
}
